docs/api-reference/webgl/texture.md

```diff
@@ -181,6 +180,31 @@ The following WebGL APIs are called in the function
 [gl.copyTexImage2D](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/copyTexImage2D), [gl.bindFramebuffer](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/bindFramebuffer)
 
 
+### copySubFramebuffer
+
+Updates defined area of a two-dimensional texture image or cube-map texture image with pixels from the current framebuffer (rather than from client memory). (gl.copyTexSubImage2D wrapper)
```

Comment:
Nits:
* "Updates defined area" => "Copies pixels from the current framebuffer into the specified area of a ..."
* "current framebuffer" => "this framebuffer"?